app/merginapi.cpp

```diff
@@ -1407,6 +1415,11 @@ void MerginApi::finalizeProjectUpdate( const QString &projectFullName )
   {
     QString projectNamespace, projectName;
     extractProjectName( projectFullName, projectNamespace, projectName );
+
+    // remove download in progress file
+    if ( !QFile::remove( InputUtils::downloadInProgressFilePath( transaction.projectDir ) ) )
+      InputUtils::log( "sync " + projectFullName, QStringLiteral( "Failed to remove download in progress file for project name %1" ).arg( projectName ) );
```

Comment:
I would prefer to leave the context string just as `"sync"` like elsewhere - and put the full project name to the arg()